index.html

```diff
@@ -176,10 +182,16 @@ <h2 class="heading">About Myself</h2>
           <a href="#" class="let-connect">LET'S CONNECT</a>
 
           <div class="social-links">
-            <a href="#" class="link"><i class="fa-brands fa-twitter"></i></a>
-            <a href="#" class="link"><i class="fa-brands fa-linkedin-in"></i></a>
+            <a href="https://twitter.com/Adekunle1855" class="link"
+              ><i class="fa-brands fa-twitter"></i
+            ></a>
+            <a href="www.linkedin.com/in/kareem-horlah" class="link"
+              ><i class="fa-brands fa-linkedin-in"></i
+            ></a>
             <a href="#" class="link"><i class="bx bxl-gmail"></i></a>
-            <a href="#" class="link"><i class="fa-brands fa-github"></i></a>
+            <a href="https://github.com/Kareem24" class="link"
+              ><i class="fa-brands fa-github"></i
+            ></a>
             <a href="#" class="link"><i class="fa-solid fa-hand-peace"></i></a>
```

Comment:
Although this is not mandatory, I believe this should be an `Angellist` logo and you can also create an account and add the link here